docs/personas.md

```diff
@@ -0,0 +1,31 @@
+# Personas
+## [P1] <a name="P1">Propietario</a>
+Dueño y CEO del restaurante. Como propietario es el encargado de tomar decisiones acertadas para el futuro de su negocio, además de gestionar el personal. 
+
+## [P2] <a name="P2">Camarero</a>
+Un empleado del restaurante. Encargado de atender de forma a las mesas, servir la comida hecha por el [cocinero](#P3) y del trato con los [clientes](#P4).
+
```

Comment:
Por favor, no pongáis todo lo que se os ocurra relacionado con el negocio. Los usuarios serán los que que obtengan algún beneficio del producto. ¿Qué beneficio obtendrá el camarero de la aplicación? ¿Podrá siquiera usarla?